src/screens/InspectionCapture/index.js

```diff
@@ -146,8 +149,22 @@ export default function InspectionCapture() {
   const settingsRef = useRef();
   const openSettings = settingsRef.current?.open;
 
+  const handleFullScreen = useCallback(() => {
+    utils.log(['[Click] Camera setting: ', 'Fullscreen ', !isFullscreen]);
+    requestFullscreen();
+  }, [isFullscreen]);
+
   const controls = [
-    { disabled: cameraLoading, ...Controls.SettingsButtonProps, onPress: openSettings },
+    !isSettingEnabled && {
+      disabled: cameraLoading,
+      ...Controls[isFullscreen ? 'ExitFullscreenButtonProps' : 'FullscreenButtonProps'],
+      onPress: handleFullScreen,
```

Comment:
No need to declare and call `handleFullScreen` function. You can directly call `requestFullscreen` function here.

Reply:
Also you should make a condition to render `<Settings ref={settingsRef} settings={settings} />` component as per `isSettingEnabled` value...